configs/urban-under-exploration-study.nrel-op.json

```diff
@@ -0,0 +1,71 @@
+{
+    "version": 1,
+    "ts": 1655143472,
+    "server": {
+        "connectUrl": "https://uue-openpath.nrel.gov/api/",
+        "aggregate_call_auth": "user_only"
+    },
```

Comment:
I do think we should have the `opcode` section - some configs don't have `opcode` for backwards compat, but going forward, we want to make sure that new configs have an explicit indication of the type of opcode generated. 
We can set up subgroups for `test` and `default, similar to CA e-bike.

To avoid the `unknown subgroup` error, we need to specify the subgroup as a query param - e.g. `sub_group=...` with the URL.

Reply:
I made those changes, and confirmed that `http://localhost:3274/?study_config=urban-under-exploration-study&sub_group=default` and `http://localhost:3274/?study_config=urban-under-exploration-study&sub_group=test` both load without the error

Reply:
I didn't realize until I looked at this, but the name of the config file needs to be the name of the host.
You need to rename the file to `uue-openpath`, similar to the others.